doc/conf/Doxyfile.in

```diff
@@ -55,9 +55,9 @@ MAX_INITIALIZER_LINES  = 30
 SHOW_USED_FILES        = YES
 SHOW_FILES             = YES
 SHOW_NAMESPACES        = YES
-FILE_VERSION_FILTER    = "@DIR_SCRIPT@/doxygen_version_extract.sh @PROJECT_SOURCE_DIR@"
 CITE_BIB_FILES         =
 INHERIT_DOCS           = YES
+USE_MATHJAX            = NO
```

Comment:
Don't we use any latex in doxygen?

Reply:
We do, but Doxygen can either translate this into images using LaTeX or embed the LaTeX code to be translated by the browser using MathJax

Reply:
Technically it's already set to `NO` by default. I just want to make sure that we really do not use it. I saw that you can get a similar issue to the `$` being interpreted as math formula inside a code block in the `html` generation part if you use mathjax also for this one (in the case of this user, it's with `\(` though):
http://doxygen.10944.n7.nabble.com/Formulas-in-code-blocks-td7537.html

Reply:
I see. thanks